README.md

```diff
@@ -44,36 +42,33 @@ Set Up PostgreSQL with Docker (docker-compose.yml provided)
 
 ### 4. Environment Variables
 
-in backend root:
+In the `backend` directory:
 
 ```sh
-BLOCKSTREAM_API_URL=https://blockstream.info/testnet/api #test-api
```

Comment:
Removed the unclear (and not well formatted - it is customary to have two spaces before the "#" and the one space after it) comment. If it's important to note that this is only a test API, you should add that in the prose above (e.g. "Note that the URL below is of the testing API").